assets/other-scripts/recaptcha/index.js

```diff
@@ -34,89 +24,180 @@ function domReady( callback ) {
 	document.addEventListener( 'DOMContentLoaded', callback );
 }
 
+window.newspack_grecaptcha = window.newspack_grecaptcha || {
+	destroy,
+	render,
+	version: newspack_recaptcha_data.version,
+};
+
+const isV2 = 'v2' === newspack_recaptcha_data.version.substring( 0, 2 );
+const isV3 = 'v3' === newspack_recaptcha_data.version;
+const siteKey = newspack_recaptcha_data.site_key;
+const isInvisible = 'v2_invisible' === newspack_recaptcha_data.version;
```

Comment:
Nonblocking nitpick: Similar to the last comment, are we supporting any other v2 version? If not, can we just assume if `isV2` is true, `isInvisible` is also true?

Reply:
[Same response as above](https://github.com/Automattic/newspack-plugin/pull/3283#discussion_r1697562186)!